util/cmd/cmd.go

```diff
@@ -39,7 +39,7 @@ func AddHealthPort(pulse Pulse, healthPort int) {
 func healthHandler(pulse Pulse) func(w http.ResponseWriter, r *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
 		if err := pulse.Health(); err != nil {
-			w.WriteHeader(http.StatusServiceUnavailable)
+			w.WriteHeader(http.StatusInternalServerError)
```

Comment:
503 really doesn't make sense for unhealthy. 503 is meant for temporary downtime during maintenance, etc. In this case we have no idea if the problem is recoverable, and it's probably not intended, so 500 fits better.